File: skill-sim-project/backend/src/test/java/com/example/skillsim/controller/HealthControllerTest.java
package com.example.skillsim.controller;

import java.util.Map;
import org.junit.jupiter.api.Test;
import static org.assertj.core.api.Assertions.assertThat;

class HealthControllerTest {

    @Test
    void checkHealthReturnsStatusOk() {
        HealthController controller = new HealthController();
        Map<String, String> response = controller.checkHealth();
        assertThat(response).containsEntry("status", "ok");
    }
}

File: backend/src/test/kotlin/com/example/skillsim/controller/HealthControllerTest.kt
package com.example.skillsim.controller

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class HealthControllerTest {

    @Test
    fun `checkHealth returns status ok`() {
        val response = HealthController().checkHealth()

        assertThat(response).containsEntry("status", "ok")
    }
}
